### PR TITLE
update NOTICE file to the new rules

### PR DESCRIPTION
**Pull request by cristipogacean**
* updated the OPC Foundation license

### CODE REVIEW

**Review by dacolgit — DISMISSED** [review]

File: NOTICE [hunk 1]
```diff
@@ -21876,16 +21881,16 @@ Distribution Requirements. For any Software Deliverables you distribute, you mus
 add significant primary functionality to it in your programs;
 require distributors and external end users to agree to terms that protect it at least as much as this Agreement;
 display your valid copyright notice on your programs; and
-indemnify, defend, and hold harmless the OPC Foundation from any claims, including attorneys’ fees, related to the distribution or use of your programs.
+indemnify, defend, and hold harmless the OPC Foundation from any claims, including attorneys� fees, related to the distribution or use of your programs.
```
Comment (karok2m) [inline NOTICE]: Please review the special characters that ended up here, like �. In the `NOTICE Opc.Ua.txt` that was provided those look like `’`. Maybe an encoding problem from the viewer that this has been copied from ?

File: NOTICE [hunk 2]
```diff
@@ -21848,10 +21848,15 @@ This license governs use of the accompanying software. If you use the software,
 
 ---------------------------------------------------------
 
-OPCFoundation.NetStandard.Opc.Ua 1.4.367.42 - OTHER
+As an OPC Foundation Corporate Member, Microsoft distributes any code dual licensed under the RCL or GPLv2 license, under the RCL license.
 
+---------------------------------------------------------
+
+OPCFoundation.NetStandard.Opc.Ua.Core 1.4.368.53 - OTHER
```
Comment (karok2m) [inline NOTICE:21855]: The version of OPCFoundation NuGet packages that is used in `main` is `1.4.367.95` and not `1.4.368.53`. Should we upgrade to `1.4.368.53` version ?

File: NOTICE (hunk 1 above)
Comment (karok2m) [reply NOTICE]: Fixed, now this shows up correctly.

**Review by karok2m — APPROVED** [review]

**Review by mregen — APPROVED** [review]
:shipit:

**Pull request merged**